apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -134,11 +134,18 @@ export interface IEnvironmentVariable {
   override?: boolean; //
 }
 
+/**
+ * Options that are only used when the NPM package manager is selected.
+ * @public
```

Comment:
Let's make this consistent with `IPnpmOptionsJson` and `IYarnOptionsJson`.

```suggestion
 * Part of IRushConfigurationJson.
 * @internal
```